[12.0][ADD] stock_request_mrp

matt454357 opened the pull request:
This is a backport from PR #864

This module allows for users to be able to display manufacturing orders that have been created as a consequence of Stock Requests.

HviorForgeFlow committed:
[13.0][ADD] stock_request_mrp

HviorForgeFlow committed:
[ADAPT] stock_request: Add stock_request_mrp installation at settings

HviorForgeFlow committed:
[ADD] post_init_hook to link SR with MO

HviorForgeFlow committed:
fixup! stock_request test + hook

rousseldenis commented:
@matt454357 Thank you for your PR. Could you just keep commit history by cherry-pick and fix travis?

matt454357 committed:
[12.0][ADD] stock_request_mrp

This is a backport from PR #864

This module allows for users to be able to display manufacturing orders that have been created as a consequence of Stock Requests.

matt454357 commented:
@rousseldenis, Have I done the cherry-picking correctly?  Do I need to increase code coverage?

HviorForgeFlow commented:
@matt454357 I did some slightly changes on stock.request model, changing production_ids from One2Many to Many2Many otherwise filters Manufacturing Order on Stock Request won't work.
